sympy/functions/special/polynomials.py

```diff
@@ -1025,8 +1025,6 @@ class hermite(OrthogonalPolynomial):
 
     """
 
-    _ortho_poly = staticmethod(hermite_poly)
-
```

Comment:
Not sure I understand this

Reply:
A consequence of the changes was that functions cannot import at top level from polys (but the other way around). So hermite_poly cannot be available at creation time.

Maybe it could be solved by a property method?

Reply:
On the higher level I *think* would want the polys module to precede anything in functions. Maybe that way around makes less sense though.

Reply:
> On the higher level I _think_ I would want the polys module to precede anything in functions. Maybe that way around makes less sense though.

Reply:
Yes, I sort of expected that poly would be imported from functions and not the other way around. It can of course be arbitrarily chosen, one just have to see the consequences.